packages/fxa-payments-server/src/index.scss

```diff
@@ -1,4 +1,15 @@
-@import '../../fxa-content-server/app/styles/main';
+@import '~normalize.css';
+@import '../../fxa-content-server/app/styles/variables';
+@import '../../fxa-content-server/app/styles/mixins';
+@import '../../fxa-content-server/app/styles/breakpoints';
+@import '../../fxa-content-server/app/styles/base';
+@import '../../fxa-content-server/app/styles/layout';
+@import '../../fxa-content-server/app/styles/modules/tooltip';
+@import '../../fxa-content-server/app/styles/modules/input-row';
+@import '../../fxa-content-server/app/styles/modules/button-row';
+@import '../../fxa-content-server/app/styles/modules/settings';
+@import '../../fxa-content-server/app/styles/modules/avatar';
+@import '../../fxa-content-server/app/styles/state';
```

Comment:
I'm not 100% sure I got all the required styles. (Is there an easy way to check?) .. 

The reason to expand it out is because there's a new style loader that conflicts with how content-server `url()`s work, hence the copies of some svgs.

Reply:
I did miss some. more coming